### DISCUSSION TITLE
V7 useForm missing property errors

### DISCUSSION THREAD

**Discussion by dqtkien** [react-hook-form/react-hook-form#4671] [Q&A] [answered]
Hi, I'm using React-Hook-Form v6.15.5 and everything is fine. Today, I update to v7.0.4 and get this error.

Here is my current code:

`type ILoginForm = {
  username: string;
  password: string;
};`

` const { handleSubmit, control, errors, reset } = useForm<ILoginForm>({
    defaultValues,
    resolver: yupResolver(LoginSchema),
  });
`
And get this error with "errors" field : `Property 'errors' does not exist on type 'UseFormReturn<ILoginForm>'.`

Please let me how to fix it or it's a bug because when I search the docs (example [link](https://react-hook-form-website-git-v7-bluebill1049.vercel.app/api/useform/seterror)), the error property is still there.

**Comment by dqtkien** [comment, marked as answer]
Sorry. Got the migration v6 to v7 [link](https://react-hook-form-website-git-v7-bluebill1049.vercel.app/migrate-v6-to-v7). Please close this thread.

**Reply by bluebill1049** [reply]
all good, keep this an answer so others can find it useful too. 👍
